.env.example

```diff
@@ -0,0 +1,17 @@
+SESSION_SECRET=hidigital
+PORT=3000
+TZ=America/Sao_Paulo
+
+DB_HOST=0.0.0.0
+DB_USER=root
+DB_PASSWORD=root
+DB_NAME=superformula
+DB_PORT=3306
+DB_PORT_TEST=3307
+
+AWS_ACCESS_KEY_ID=AKIA2CWT2RQ2QUL76R6A
```

Comment:
How would you handle this in a production app?

Reply:
We need to have a .env file for each environment, this is just the example to allows you to start the project